<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTaskDialog</class>
 <widget class="QDialog" name="NewTaskDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Новое задание</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="task_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя задания:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Директория задания:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameTask">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="task_path">
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="TaskButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
